test/jit/test_symbolic_shape_analysis.py

```diff
@@ -120,74 +116,6 @@ def neg_to_one(li):
         if_out = next(foo.graph.findNode("prim::If").outputs())
         self.assertEqual(neg_to_one(if_out.type().symbolic_sizes()), [-1, 3, -1, -1])
 
-    def test_unary_shape_functions(self):
-        unary_ops = [
-            torch.nn.functional.hardtanh,
-        ]
-        for fn in unary_ops:
-            t = torch.jit.trace(fn, (torch.rand([4, 4])))
-            ten_input = next(t.graph.inputs())
-            ten_input.setType(ten_input.type().with_sizes([2, 2]))
-            torch._C._jit_pass_propagate_shapes_on_graph(t.graph)
-            self.assertEqual(next(t.graph.outputs()).type().symbolic_sizes(), [2, 2])
-
-    def test_unary_shape_fns_inplace(self):
-        def mul_inplace(x: torch.Tensor):
-            y = x.mul_(2)
-            return y
-
-        unary_ops = [
-            mul_inplace
-        ]
-        for fn in unary_ops:
-            # t = torch.jit.trace(fn, torch.rand([4, 4]))  # For some reason tracing is erroring out.
-            t = torch.jit.script(fn)
-            ten_input = next(t.graph.inputs())
-            ten_input.setType(ten_input.type().with_sizes([2, 2]))
-            torch._C._jit_pass_propagate_shapes_on_graph(t.graph)
-            self.assertEqual(next(t.graph.outputs()).type().symbolic_sizes(), [2, 2])
-
-    def test_binary_shape_functions(self):
-        binary_ops = [
-            operator.__mul__,
-            operator.__truediv__,
-            operator.__gt__,
-            operator.__add__,
-        ]
-
-        for fn in binary_ops:
-            size_1 = [1, 4, 8]
-            size_2 = [4, 1, 8]
-            t = torch.jit.trace(fn, (torch.rand([4]), torch.rand([4])))
-            inputs = list(t.graph.inputs())
```

Comment:
These aren't really per-op testing like the other stuff, it's testing the shared infrastructure between inplace/functional operators, could we just leave these here ? E.g. we don't have a good set of exhaustive inputs or anything like in the other test 
 
I don't really like moving things around without a good reaon because it makes tracking the blame harder